server/src/test/java/com/objectcomputing/checkins/services/TestContainersSuite.java

```diff
@@ -36,6 +37,9 @@ public abstract class TestContainersSuite implements RepositoryFixture, TestProp
     @Inject
     private Flyway flyway;
 
+    @Value("${aes.key}")
+    protected String key;
+
```

Comment:
I'm confused by this...

Reply:
This is runtime injection of the aeskey which is required for the custom query